code/modules/mob/living/brain/living_brain.dm

```diff
@@ -18,31 +18,28 @@
 	return ..()
 
 /mob/living/brain/say_understands(other)//Goddamn is this hackish, but this say code is so odd
+	if(!ismob(other))
+		return ..()
 	if(isAI(other))
 		if(!(container && istype(container, /obj/item/mmi)))
-			return 0
-		else
-			return 1
+			return FALSE
+		return TRUE
```

Comment:
Annother place where Github doesn't want to allow a suggestion, sadly.

```dm
		return in_mmi
```

And the other ones below as well.